operator/docs/enhancements/ruler_support.md

```diff
@@ -98,142 +124,218 @@ type LokiStackComponentStatus struct {
 }
 ```
 
-#### LokiRule definition
+#### AlertingRule definition
```

Comment:
Should `AlertingRule` and `RecordingRule` be prefixed with `Loki`? They will not conflict with Prometheus (should the other operator ever decide to split the rules up as well) because of the group, but this might be more convenient / visible for users.

Reply:
The proposal started to use `Loki` as a prefix, but toward refinement I realized that the typical kubernetes naming follows the GroupVersion approach. Thus when we speak of `AlertingRule` we mean `alertingrule.loki.grafana.com/v1beta1` and similar for `RecordingRule` we speak of `recordingrule.loki.grafana.com/v1beta1`. Ergo prefixes make names longer and double the amount of `loki` references in the full name.

Reply:
This might be just me, but I use the "unqualified names" of Kubernetes types a lot, so I would just write out "alertingrule" when using kubectl. As I said this is just for convenience, so it's fine the way it currently is.